enclave-manager/web/src/components/CopyButton.tsx

```diff
@@ -15,7 +15,7 @@ export const CopyButton = ({ valueToCopy, text, ...buttonProps }: CopyButtonProp
       const v = typeof valueToCopy === "string" ? valueToCopy : valueToCopy();
       navigator.clipboard.writeText(v);
       toast({
-        title: `Copied '${v}' to the clipboard`,
+        title: `Copied url to the clipboard`,
```

Comment:
It's not always a url that is copied - I think this is better exposed as a prop.

Reply:
Done